docker/app/Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM sillsdev/web-languageforge:base-app-latest
+FROM sillsdev/web-languageforge:app-base-latest
```

Comment:
before this PR gets merged, it would be great to ensure that this image is pushed and exists so that the master build doesn't break.

Reply:
yep, already did that in early commits on this branch (https://github.com/sillsdev/web-languageforge/pull/937/commits/1c4b8e772cb8f759e8bb1d17f48a2b10810d24da).

My hope is that most devs are not powerwashing, the CI environment is the same so everyone should already have this old image anyway.  Once this merges into master all will be addressed in a normal CI fashion.